File: Java/UberBlack.java
import java.util.ArrayList;
import java.util.Map;

class UberBlack extends Car{
    
    Map<String, ArrayList<String,Integer>> typeCardAccepted;
    ArrayList<String> seatsMaterial;

    public UberBlack(String pLicense, Account pDriver, String pBrand, String pModel,Map<String, ArrayList<String,Integer>> typeCardAccepted,
    ArrayList<String> seatsMaterial){
        super(pLicense, pDriver);
        this.typeCardAccepted = typeCardAccepted;
        this.seatsMaterial = seatsMaterial;
    }
}

File: PHP/uberBlack.php
<?php
require_once 'Car.php';
class UberBlack extends Car{

    public $typeCardAccepted;
    public $seatsMaterial;

    public function __constructor($pLicense, $pDriver, $pTypeCardAccepted, $pSeatsMaterial){
        parent:: __constructor($pLicense, $pDriver);
        $this->typeCardAccepted = $pTypeCardAccepted;
        $this->seatsMaterial = $pSeatsMaterial;
    }

}